### ISSUE TITLE
Buttons must have discernible text

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-next#1906]
We have this a11y violation in the following paths

`/demos/CardView/examples-full/` where the code looks like
```html
<button class="pf-c-button pf-m-plain pf-c-toolbar__filter-toggle">
  <i class="fas fa-filter" aria-hidden="true"></i>
</button> 
```

and again in
`/components/Toolbar/examples-full/` where it's another occurrence of the same component

```html
<button class="pf-c-button pf-m-plain pf-c-toolbar__filter-toggle">
  <i class="fas fa-filter" aria-hidden="true"></i>
</button>
```

Is it any more complicated than adding an aria-label to the button? Let me know either way, and I'm happy to send a fix.

**Comment by mcoker** [comment]
> Is it any more complicated than adding an aria-label to the button? Let me know either way, and I'm happy to send a fix.

That's it!
